### PR TITLE
fix: -ws option causes layout failures.

### PR DESCRIPTION
**Pull request by mikeal**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test": "say sorry",
-    "start": "hugo server -ws .",
+    "start": "hugo server .",
```
Comment (vmx) [inline package.json:8]: This should simply be `hugo server` without the `.`.

**Review by vmx — COMMENTED** [review]
For me it doesn't make any difference in the layout. But I'm on Linux, so it might be an MacOS thing. But if it doesn't make any difference, it should be fine. Please simplify the command as mentioned inline.

**Comment by olizilla** [comment]
I think the big problem here is that `hugo` is expected to be a global dep, but there are breaking changes to the expected project layout and the command arguments, which is dull. I'll raise a PR to pin hugo as a local dev dep.

**Comment by olizilla** [comment]
This was fixed in https://github.com/ipld/website/pull/54 by including `hugo` in the project dependencies and using the version local to the project rather than hoping the user has a compatible version installed.

**Pull request closed without merging**